rust/Dockerfile

```
@@ -78,6 +78,11 @@ RUN apt-get update && \
     mkdir -p /usr/share/hyperlane && chmod 1000 /usr/share/hyperlane && \
     mkdir -p /data && chown -R 1000 /data
 
+# Create a proper home for the non-root user and Aleo resources
+ENV HOME=/home/hyperlane
+RUN mkdir -p /home/hyperlane/.aleo/resources && \
+    chown -R 1000:1000 /home/hyperlane
```

dont create a separate step for this, instead just add to the end of the runtime image setup
```suggestion
    mkdir -p /usr/share/hyperlane && chmod 1000 /usr/share/hyperlane && \    
    mkdir -p /data && chown -R 1000 /data && \
    mkdir -p /home/hyperlane/.aleo/resources && \
    chown -R 1000:1000 /home/hyperlane
```

---

Okay sir 🫡